README.md

```diff
@@ -157,29 +206,41 @@ docker run \
     docker.greenkeytech.com/svtserver
 ```
 
-Once complete, you should have a JSON file for each audio file you generated. This JSON file contains the **word confusion lattice** that Discovery searches for your target phrases.
+Once complete, you should have a JSON file for each audio file you generated (e.g. `test.json` for `test.wav`). This JSON file contains the **word confusion lattice** that Discovery searches for your target phrases.
+
+These JSON files can be used directly with the Discovery Engine as [shown here](https://transcription.greenkeytech.com/discovery-1890af/deploying/#6-run-the-scribe-discovery-engine-on-a-file). The example directories provide guidance on how to send these files to discovery in the `send_transcript_to_discovery.sh` script.
+
+For the 'room_dialing' example, `send_transcript_to_discovery.sh` contains:
+```bash
+curl -X POST http://localhost:1234/discover \
+     -H "Content-Type: application/json" \
+     -d '{"transcript": "dial one eight"}'
+```
+
+
+## Full Documentation
+Our full [documentation](https://transcription.greenkeytech.com/discovery-1890af/) provides many more in-depth descriptions, explanations and examples.
 
-These JSON files can be used directly with the Discovery Engine as [shown here](https://transcription.greenkeytech.com/discovery-1890af/deploying/#getting-started). The example directories provide guidance on how to send these files to discovery in the `send_transcript_to_discovery.sh`.
 
-## Contributing
+# Contributing
 
-### Code of Conduct
+## Code of Conduct
 
 Please make sure you read and observe our [Code of Conduct].
 
-### Pull Request process
+## Pull Request process
 
 1. Fork it
```

Comment:
they're all numbered `1.`

Reply:
If you do a bunch of ones, markdown automatically replaces with ascending numbers.